src/Makefile.am

```diff
@@ -437,6 +437,7 @@ endif
 
 TESTS_ENVIRONMENT = \
 	srcdir="$(srcdir)" \
+	top_srcdir="$(srcdir)" \
```

Comment:
This has to be `top_srcdir=top_srcdir`, or `top_srcdir=srcdir` is correct?

Reply:
I think you're right, fixed.